tutorials/getting_started.ipynb

```diff
@@ -297,22 +488,44 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 13,
+   "execution_count": 39,
    "metadata": {},
    "outputs": [],
    "source": [
-    "new_obj = pfbi.create_in_folder(r\"Library\\Dynamic Models\", \"dummy.BlkDef\", overwrite=True)\n",
+    "new_obj:BlkDef = pfp.create_in_folder(\"dummy.BlkDef\", r\"Library\\Dynamic Models\", overwrite=True)\n",
     "# Or equivalent:\n",
-    "models_folder = pfbi.get_single_obj(r\"Library\\Dynamic Models\")\n",
-    "new_obj = pfbi.create_in_folder(models_folder, \"dummy.BlkDef\", overwrite=True)\n"
+    "models_folder:IntPrjfolder = pfp.get_single_obj(r\"Library\\Dynamic Models\")\n",
+    "new_obj:BlkDef = pfp.create_in_folder(\"dummy.BlkDef\", models_folder, overwrite=True)"
    ]
   },
   {
    "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "which will overwrite the former object (this avoids that *PowerFactory* creates *dummy(1), dummy(2), ...* every time the code is run). `overwrite=True` is the default setting."
+    "which will overwrite the former object (this avoids that *PowerFactory* creates *dummy(1), dummy(2), ...* every time the code is run). `overwrite=True` is the default setting. \n",
+    "\n",
+    "To create a directory use"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 40,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<powerfactory.DataObject <l1>\\seberlein.IntUser\\powfacpy\\powfacpy_tests.IntPrj\\Study Cases.IntPrjfolder\\New Study Case\\Cases Normal Opration</l1>>"
+      ]
+     },
+     "execution_count": 40,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "pfp.create_directory(r\"New Study Case\\Cases Normal Opration\", parent_folder=\"Study Cases\")"
```

Comment:
The [link from above](https://stackoverflow.com/questions/5078676/what-is-the-difference-between-a-directory-and-a-folder) says that directory and folder can be used interchangeably. (There is a difference, but that is not relevant in this case) That would mean that create_folder and create_directory mean the same. 

If you have a look [here](https://unix.stackexchange.com/questions/131561/what-is-the-difference-between-path-and-directory), in short, the difference between path and directory is

"Directory is where you are. Path is how to get there."

Reply:
hey @Sciemon, just started using the new version of powfacpy, seems like the structure really improved!

I stumbled over the create_directory again and I find it confusing, because of the reasons given above.  what do you think about renaming it to

create_directory_path() or create_folder_path() ?

I did some research and apparently that is how you would call what you mean.

Reply:
"I did some research and apparently that is how one would call what you mean."
After having a look at the links this is not apparent to me and it is a bit hard for me to understand why you seem to be so convinced by your reasoning. 

For me the strongest argument to use "directory", as already pointed out in my first comment, is still that this is the term used in Windows commands/Python/Matlab/Julia to do pretty much exactly what the method does in the windows file system instead of the PF datamanager:

- Windows: mkdir 
- Python: os.makedirs
- Matlab: mkdir
- Julia has mkdir (but also mkpath)

Linux also has mkdir. 

powfacpy users are used to Windows so they are used to its terminology.

As "create" is more common than "make" in powfacpy, "create_directory" is the most reasonable name for the method. 

I have never seen anything like "create_directory_path" or "create_folder_path" anywhere. If you search online for those words, results point to mkdir.